src/main/python/model/postbuilder.py

```diff
@@ -0,0 +1,117 @@
+import logging
+import os
+import sys
+
+from qtpy.QtWidgets import QDialog, QFileDialog
+
+from model.iir import Gain
+from model.minidsp import HDXmlParser, pad_with_passthrough
+from model.preferences import BEQ_DOWNLOAD_DIR
+from ui.postbuilder import Ui_postbuilder
+
+logger = logging.getLogger('postbuilder')
+
+
+class CreatePostDialog(QDialog, Ui_postbuilder):
```

Comment:
there is nothing here to say what we're creating, I guess it's an avs post but not 100% sure without running it

Reply:
I'll go through and be more specific that it's AVS related.  Good suggestion.  I was in a "this is for Aron and he knows what it is" mindset.

Reply:
I pushed a new commit addressing all of your comments.